stglib/iq.py

```diff
@@ -91,142 +96,174 @@ def read_iq(filnam):
                     ds[k].attrs["units"] = iqmat["Data_Units"][k].replace("/s", " s-1")
             elif "_2_" in k or "_3_" in k:
                 ds[k] = xr.DataArray(
-                    iqmat[k][0:timelen, :], dims=("time", "cell_across")
+                    iqmat[k][0:timelen, :], dims=("time", "bin_across")
                 )
                 if k in iqmat["Data_Units"]:
                     ds[k].attrs["units"] = iqmat["Data_Units"][k].replace("/s", " s-1")
             elif "_0_" in k or "_1_" in k:
-                ds[k] = xr.DataArray(
-                    iqmat[k][0:timelen, :], dims=("time", "cell_along")
-                )
+                ds[k] = xr.DataArray(iqmat[k][0:timelen, :], dims=("time", "bin_along"))
                 if k in iqmat["Data_Units"]:
                     ds[k].attrs["units"] = iqmat["Data_Units"][k].replace("/s", " s-1")
-            elif "FlowData_Vel" in k or "FlowData_SNR" in k:
+            elif "FlowData_Vel" in k and "XYZ" not in k or "FlowData_SNR" in k:
```

Comment:
This logic is a little confusing, mixing ands and ors. Suggest adding parentheses for clarity.

Reply:
Ok, I ended up splitting those up:

`            elif "FlowData_SNR" in k:
                ds[k] = xr.DataArray(iqmat[k][0:timelen, :], dims=("time", "velbeam"))
                ds[k].attrs["units"] = iqmat["Data_Units"][k]
            elif "FlowData_Vel" in k and "XYZ" not in k:
                ds[k] = xr.DataArray(iqmat[k][0:timelen, :], dims=("time", "velbeam"))
                if k in iqmat["Data_Units"]:
                    ds[k].attrs["units"] = iqmat["Data_Units"][k].replace("/s", " s-1")`